### DISCUSSION TITLE
Application shuts down over night on one PC

### DISCUSSION THREAD

**Discussion by mikehd1959** [keepassxreboot/keepassxc#9359] [Q&A]
I have keepassxc installed on two Windows 10 PCs at my home. One is set to sleep, the other is on 24x7 to support client backups. The one that sleeps is the PC where keepassxc will shutdown overnight. The other PC never has this issue.

The PC that sleeps also does not require a password upon wake up. Could this behavior of keepassxc be intentional so that it forces a login when the PC is awakened?

**Comment by droidmonkey** [comment]
We don't have any hooks into the PC login methods. Perhaps this has something to do with RAM swap and it causing a crash.

**Reply by mikehd1959** [reply]
I took a look at settings to see if there were methods for enabling a logfile, and in appdata for an existing logfile, but found no mentions. How would I enable logging? I did go thru the Windows Event Viewer looking for application crashes related to keepassxc but found nothing.

**Reply by droidmonkey** [reply]
Is keepassxc the only app that closes on sleep resume? This is not normal behavior.

**Reply by mikehd1959** [reply]
It is the only one that I've noticed. I usually have a bunch of background apps running with icons in the notification tray. keepassxc is the only one that routinely closes.
